src/core/Drawer/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { makeStyles, Theme } from "@material-ui/core";
+
+interface StyleProps {
+  anchor: string | undefined;
+}
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+  },
+  content: {
+    padding: "1rem",
+    width: (props: StyleProps) =>
+      props.anchor === "top" || props.anchor === "bottom"
+        ? "100%"
+        : "fit-content",
+    color: theme.palette.text.primary,
+    background: theme.palette.background.paper,
+    textAlign: "center",
+  },
+  closeIcon: {
+    float: "right",
+    width: "3rem",
```

Comment:
```suggestion
    width: "fit-content",
```

Reply:
Tried to made this change, the iconbutton was still in oval shape.
![Screenshot from 2021-08-25 12-09-39](https://user-images.githubusercontent.com/40661238/130739667-c710ac92-c3a5-41ec-af89-c32f053be542.png)